### ISSUE TITLE
FEATURE REQUEST: Consider parameter fetch_compute_hosts (default True) to allow only DB inventories

### ISSUE THREAD

**Issue by jeliker** [oracle/oci-ansible-collection#40]
I would like to have a parameter such as `fetch_compute_hosts` that I could set `False` to combine with `fetch_db_hosts=True` and produce a DB-only inventory.

In some cases, we prefer inventory of only DB since OCI DB Systems are often managed in unique ways (i.e. unlike management of general Compute) to ensure we are respecting the "managed server" principles.

**Comment by manojmeda** [comment]
@jeliker Thanks for raising this issue. Agreed that it is definitely useful. We are checking this and we will update you soon.

**Comment by AmeyaLokre** [comment]
Hello @jeliker , we will be working on this feature request. As a workaround for now, you can add a filter in the config profile to set the lifecycle_state to `AVAILABLE` as compute instances have a lifecycle_state of `RUNNING`.

```
plugin: oracle.oci.oci

enable_parallel_processing: yes

filters:
  - lifecycle_state: "AVAILABLE"

fetch_db_hosts: yes

**Comment by jeliker** [comment]
Clever work-around! Thank you.

**Comment by bsushant-athena** [comment]
Hey @jeliker, we have added a fix for this issue in our latest release 2.15.0. Please check that out
